space/internal/components/aclnotifications/notifications.go

```diff
@@ -99,13 +108,34 @@ func (n *aclNotificationSender) AddRecords(acl list.AclList,
 			spaceId:       spaceId,
 			aclId:         acl.Id(),
 			accountStatus: accountStatus,
+			spaceName:     spaceName,
 		})
 		if err != nil {
 			logger.Errorf("failed to add acl record, %s", err)
 		}
 	}
 }
 
+func (n *aclNotificationSender) getSpaceName(spaceId string) string {
+	records, _, err := n.objectStore.Query(database.Query{
```

Comment:
This should work, though I would also make sure that the layout is "spaceView" (add another check), so we would not get some other object which in theory can use the same relation